### PR TITLE
update 2 database books

### PR DESCRIPTION
**Pull request by hackerj122**
## What does this PR do?
Add resource(s)

## For resources
database
### Description
added the 2 books pdf 
### Why is this valuable (or not)?
the books are awesome and easily understandable. 
### How do we know it's really free?
The book is available freely in pdf format
### For book lists, is it a book? For course lists, is it a course? etc.
Its a books for book list
## Checklist:
- [1 ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [1 ] Search for duplicates.
- [1 ] Include author(s) and platform where appropriate.
- [1 ] Put lists in alphabetical order, correct spacing.
- [1 ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -238,6 +238,8 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Readings in Database Systems, 5th Ed.](http://www.redbook.io)
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
+* [DATABASE SYSTEMS The Complete Book Second Edition](https://people.inf.elte.hu/miiqaai/elektroModulatorDva.pdf)
```
Comment (davorpa) [inline books/free-programming-books-subjects.md:241]: Please, read page 4-7.

> NOTICE
> This work is protected by U.S. copyright laws and is provided solely for the use of college instructors in reviewing course materials for classroom use. Dissemination or sale of this work, or any part (including on the World Wide Web), is not permitted.

> http://infolab.stanford.edu/~ullman/dscb.html

@eshellman, Should we accept this kind of works, which are only for instructors (not sure if partial free, I think, reading official website)? Discarded in the past: #6410 #6349

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -238,6 +238,8 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
...
+* [DATABASE MANAGEMENT SYSTEM (DBMS)](https://mrcet.com/downloads/digital_notes/ECE/III%20Year/DATABASE%20MANAGEMENT%20SYSTEMS.pdf)
```
Comment (davorpa) [inline books/free-programming-books-subjects.md:242]: Who is the author?

File: books/free-programming-books-subjects.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-subjects.md:241]: 1. This is clearly not an authorized distribution.
2. ALL CAPS is not appropriate for catalog entries except for acronyms and sometimes proper names.

**Comment by eshellman** [comment]
This appears not to be an authorized distribution. Please see CONTRIBUTING.md guidelines.

**Pull request closed without merging**